sumologic/resource_sumologic_data_forwarding_rule.go

```diff
@@ -33,12 +33,10 @@ func resourceSumologicDataForwardingRule() *schema.Resource {
 			"enabled": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Default:  true,
```

Comment:
This looks like a change in behavior. A rule will be disabled by default now? This might be a breaking change. The good part is we are scheduled for a major release if this is a breaking change it should be fine. We should document the new behavior.

Reply:
Amm, no it's not a breaking change. Because in the [API](https://api.sumologic.com/docs/#operation/createDataForwardingRule), we don't enforce any default value for `enabled`, which means that it'll be treated as `false` if left blank. 

Here, I'm correcting a mistake from my previous PR .. and and bringing it in-line with the API.